<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PNtrianglesClass</class>
 <widget class="QMainWindow" name="PNtrianglesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PNtriangles</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyViewer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>631</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="gl_draw">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GL Draw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="custom_draw">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>110</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom Draw</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLvl">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="fill">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Figure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pn_map">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PN Map</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_lvl_map">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>250</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_map">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_add_map">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>300</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="read_lvl">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>350</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyViewer</class>
   <extends>QWidget</extends>
   <header>myviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PNtriangles.qrc"/>
 </resources>
 <connections/>
</ui>
